lib/options.json

```diff
@@ -241,6 +241,12 @@
         },
         {
           "type": "boolean"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
items, can be not only strings

Reply:
According to this, I believe they are only string https://www.npmjs.com/package/open#usage

Reply:
`{wait: true}`